### ISSUE TITLE
Pivot table is hiding information

### ISSUE THREAD

**Issue by kennyrogerson** [metabase/metabase#45708]
### Describe the bug

When I create a pivot table without filtering and summarized by value divided into 4 fields, the last level only shows 1 record. When filtering by customer, all the correct records appear
![print-pivot-table-with-filter](https://github.com/user-attachments/assets/26f5f2cf-345d-49e5-b09e-a9deea555f61)
![print-pivot-table-without-filter](https://github.com/user-attachments/assets/1ae124c5-1edc-4da7-bb57-5c4c8317569a)
![Print-editor](https://github.com/user-attachments/assets/28460d5b-32db-4e9b-b447-6a924bb8cf29)

### Expected behavior

All records appear when I don't choose any filter.
![print-pivot-table-with-filter](https://github.com/user-attachments/assets/537ea80d-d20c-4838-9448-6f7b5e91660f)

### Logs

[9e860b23-866c-42a4-b8e8-97dc31f64e02] 2024-07-17T08:21:58-03:00 INFO metabase.core 
Metabase v0.50.13 (2086968) 

Copyright © 2024 Metabase, Inc. 

Metabase Enterprise Edition extensions are NOT PRESENT.
[9e860b23-866c-42a4-b8e8-97dc31f64e02] 2024-07-17T08:21:58-03:00 INFO metabase.core Starting Metabase in STANDALONE mode
[9e860b23-866c-42a4-b8e8-97dc31f64e02] 2024-07-17T08:21:58-03:00 INFO metabase.server Launching Embedded Jetty Webserver with config:
 {:port 3000}

[log.txt](https://github.com/user-attachments/files/16264907/log.txt)

### Information about your Metabase installation

```JSON
{
  "browser-info": {
    "language": "pt-BR",
    "platform": "Win32",
    "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.23+9-post-Ubuntu-1ubuntu120.04.2",
    "java.vendor": "Ubuntu",
    "java.vendor.url": "https://ubuntu.com/",
    "java.version": "11.0.23",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.23+9-post-Ubuntu-1ubuntu120.04.2",
    "os.name": "Linux",
    "os.version": "5.4.0-187-generic",
    "user.language": "pt",
    "user.timezone": "America/Sao_Paulo"
  },
  "metabase-info": {
    "databases": [
      "postgres"
    ],
    "hosting-env": "unknown",
    "application-database": "h2",
    "application-database-details": {
      "database": {
        "name": "H2",
        "version": "2.1.214 (2022-06-13)"
      },
      "jdbc-driver": {
        "name": "H2 JDBC Driver",
        "version": "2.1.214 (2022-06-13)"
      }
    },
    "run-mode": "prod",
    "plan-alias": "",
    "version": {
      "date": "2024-07-15",
      "tag": "v0.50.13",
      "hash": "2086968"
    },
    "settings": {
      "report-timezone": null
    }
  }
}
```

### Severity

is hiding information

**Comment by bshepherdson** [comment]
I think the screenshots are in a different order than intended? It makes this report hard to follow.

It's not clear what you expected to see, and what you're seeing instead.

**Comment by paoliniluis** [comment]
closing due to non response
